libs/components/theme/src/lib/styles/_theme.scss

```diff
@@ -128,3 +128,11 @@ mark.sky-highlight-mark {
 .sky-field-status-success {
   @include mixins.sky-field-status('success');
 }
+
+.sky-isolate {
```

Comment:
These should be combined into one CSS class that's specific to the wait component. Something like `sky-wait-element-active`. If we ever need to add properties to the wait, they can be added to that class.

Reply:
Updated to use a single class.